example/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.0.1-all.zip
+networkTimeout=10000
```

Comment:
what is this property?

Reply:
good question!? I'm not sure. This all came over from the RN update tool here: https://react-native-community.github.io/upgrade-helper/?from=0.69.9&to=0.72.4#RnDiffApp-android-gradle-wrapper-gradle-wrapper.properties